app/websocket/app/main/routes.py

```diff
@@ -96,6 +96,21 @@ def index():
     return render_template("index.html")
 
 
+@main.route("/dataset-tags", methods=["GET"])
+def get_available_tags():
+    """ Returns all tags stored in this node. Can be very useful to know what datasets this node contains. """
+    available_tags = list()
```

Comment:
I think this should be a set from the beginning instead of turning it into a set

Reply:
Done! (6ace96c)